src/main/java/pl/psnc/dei/service/BatchService.java

```diff
@@ -138,7 +139,14 @@ private String getTitle(String recordId) {
 					.filter(obj -> obj.get("dc:title") != null)
 					.findFirst();
 			if (title.isPresent()) {
-				return title.get().getAsObject().get("dc:title").getAsObject().get("@value").getAsString().value();
+				JsonValue dcTitle = title.get().getAsObject().get("dc:title");
+				if (dcTitle.isObject()) {
+					return dcTitle.getAsObject().get("@value").getAsString().value();
+				} else if (dcTitle.isString()) {
```

Comment:
else is not necessary, especially the last one

Reply:
fixed